repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/SearchCriteriaRepositoryTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.ccd.definition.store.repository;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchCriteriaEntity;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {
+    SanityCheckApplication.class,
+    TestConfiguration.class
+})
+@TestPropertySource(locations = "classpath:test.properties")
+@Transactional
+public class SearchCriteriaRepositoryTest {
+
+    private static final String CASE_TYPE_REFERENCE = "CaseTypeA";
+
+    private static final String OTHER_CASE_REFERENCE = "OtherCaseReference";
+
+    @Autowired
+    private SearchCriteriaRepository searchCriteriaRepository;
+
+    @Autowired
+    private TestHelper testHelper;
+
+    private CaseTypeEntity latestCaseType;
+
+    @Before
+    public void setUp() {
+        latestCaseType = testHelper.createCaseType(CASE_TYPE_REFERENCE, CASE_TYPE_REFERENCE);
+
+        createSearchCriteriaEntity(latestCaseType);
+    }
+
+    @Test
+    public void shouldGetSearchCriteriaEntity() {
+        List<SearchCriteriaEntity> result = searchCriteriaRepository
+            .findSearchCriteriaEntityByCaseType(CASE_TYPE_REFERENCE);
+
+        assertAll(
+            () -> assertThat(result, hasSize(1)),
+            () -> assertThat(result.get(0).getCaseType(), is(latestCaseType)),
+            () -> assertThat(result, hasItem(hasProperty("otherCaseReference", is(OTHER_CASE_REFERENCE))))
+        );
+    }
+
+    @Test
+    public void shouldReturnNoSearchCriteriaResultForUnknownParameters() {
+        List<SearchCriteriaEntity> result = searchCriteriaRepository
+            .findSearchCriteriaEntityByCaseType("IncorrectCaseType");
+
+        assertAll(
```

Comment:
Dont need assertAll here ?

Reply:
done in [59c375d1](https://github.com/hmcts/ccd-definition-store-api/pull/1061/commits/59c375d172d81b375f2fe43039955b022afd906e)